Movies/Application/DetailScreen/Presenter/DetailMoviePresenter.swift

```diff
@@ -0,0 +1,56 @@
+// DetailMoviePresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+protocol DetailMovieViewProtocol: AnyObject {
+    func success()
+    func failure(error: Error)
+    func showWebViewController(viewController: WebViewController)
+}
+
+protocol DetailMoviePresenterProtocol: AnyObject {
+    func fetchData(_ movieId: Int)
+    func showWebView()
+    var movieDetail: MovieDetail? { get set }
+    var movieID: Int { get set }
+}
+
+final class DetailMoviePresenter: DetailMoviePresenterProtocol {
+    // MARK: - Public Properties
+
+    var movieDetail: MovieDetail?
+    private weak var view: DetailMovieViewProtocol?
+    var networkService: MovieAPIServiceProtocol!
```

Comment:
```suggestion
    var movieAPIService: MovieAPIServiceProtocol!
```
по тз сказал же переименовать сервис